examples/camera-app/linux/include/clusters/push-av-stream-transport/push-av-stream-manager.h

```diff
@@ -52,6 +55,8 @@ class PushAvStreamTransportManager : public PushAvStreamTransportDelegate
     void Init();
     void SetMediaController(MediaController * mediaController);
     void SetCameraDevice(CameraDeviceInterface * cameraDevice);
+    void SetOnRecorderStoppedCallback(std::function<void(uint16_t, PushAvStreamTransport::TransportTriggerTypeEnum)> cb) override;
```

Comment:
It is not clear what the uint16_t arguments may represent. Please document above some of these details.